src/optimisation/Optimisation.jl

```diff
@@ -1,324 +1,327 @@
 module Optimisation
 
 using ..Turing
+using ..OptimisationCore: ModeResult, MLE, MAP, OptimLogDensity, OptimizationContext,
+    ModeEstimator, variable_names
+using NamedArrays: NamedArray
+using Optimization
+using OptimizationOptimJL: LBFGS, IPNewton
+using DocStringExtensions: TYPEDFIELDS
 using Bijectors
 using Random
-using SciMLBase: OptimizationFunction, OptimizationProblem, AbstractADType, NoAD
+using SciMLBase: OptimizationFunction, OptimizationProblem, solve
+using ADTypes: AbstractADType
 
 using Accessors: Accessors
-using DynamicPPL
-using DynamicPPL: Model, AbstractContext, VarInfo, VarName,
-    _getindex, getsym, getfield,  setorder!,
-    get_and_set_val!, istrans
+using DynamicPPL: Model, DefaultContext, LikelihoodContext
```

Comment:
For stuff from `DynamicPPL` it's "less" important to qualify the names, _but_ I'm still preferential to doing so (at least for symbols not explicitly exported)

Reply:
Agreed, done.